pdata/internal/metrics.go

```diff
@@ -183,6 +183,62 @@ func (at MetricAggregationTemporality) String() string {
 	return otlpmetrics.AggregationTemporality(at).String()
 }
 
+// EmptyMetricDataPointFlags defines how a metric aggregator reports aggregated values.
+// It describes how those values relate to the time interval over which they are aggregated.
+//
+// This is a reference type, if passed by value and callee modifies it the
+// caller will see the modification.
+//
+// Must use NewEmptyMetricDataPointFlags function to create new instances.
+// Important: zero-initialized instance is not valid for use.
+type EmptyMetricDataPointFlags struct {
```

Comment:
s/EmptyMetricDataPointFlags/MetricDataPointFlagsStruct?

If I understand, this is a temporary name because of the name conflict, correct?

Reply:
Ya temporary. In a future PR we'd switch everything back to the names being used today.

Reply:
Please use "MetricDataPointFlagsStruct" name.

Reply:
Done